sos/report/plugins/foreman_proxy.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (C) 2015 Red Hat, Inc., Pavel Moravec <pmoravec@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, RedHatPlugin, DebianPlugin,
+                                UbuntuPlugin)
+
+
+class ForemanProxy(Plugin):
+
+    short_desc = 'Foreman SmartProxy systems management'
+
+    plugin_name = 'foreman_proxy'
+    profiles = ('sysmgmt',)
+    packages = ('foreman-proxy', )
+
+    option_list = [
+        ('proxyfeatures', 'list features of the smart proxy', 'slow', False),
+    ]
```

Comment:
Is it intentional that this option is also left in the foreman plugin? Or should it be removed there and left only in this new proxy plugin? If the latter we'll need to update the test in tests/product_tests/foreman/foreman_tests.py.